### PR TITLE
Info about verifying downloads

### LINKED ISSUE #35189: Add information about verifying the download file
The doc in dotnet/core#8420 is documenting how to verify the downloaded file on mac/linux/windows. These overview install docs should add similar information.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 1eeab451-820f-de42-983e-4d0d5764f6ab
* Version Independent ID: d11d0d14-8a03-50e4-5c00-9a4417ad4d0a
* Content: [Install .NET on Windows - .NET](https://learn.microsoft.com/en-us/dotnet/core/install/windows?tabs=net70)
* Content Source: [docs/core/install/windows.md](https://github.com/dotnet/docs/blob/main/docs/core/install/windows.md)
* Product: **.net**
* Technology: **dotnet-installation**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

---
[Associated WorkItem - 87513](https://dev.azure.com/msft-skilling/Content/_workitems/edit/87513)

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

Add information about how to verify the checksum of downloaded binaries

Fixes #35189

@leecow please see my note here: https://github.com/dotnet/core/pull/8420#issuecomment-1601567352

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/install/linux-scripted-manual.md](https://github.com/dotnet/docs/blob/a8768abfe9157bf1aedea468edef3022bc9b4fa7/docs/core/install/linux-scripted-manual.md) | [Install .NET on Linux by using an install script or by extracting binaries](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-scripted-manual?branch=pr-en-us-35911) |
| [docs/core/install/macos.md](https://github.com/dotnet/docs/blob/a8768abfe9157bf1aedea468edef3022bc9b4fa7/docs/core/install/macos.md) | [Install .NET on macOS](https://review.learn.microsoft.com/en-us/dotnet/core/install/macos?branch=pr-en-us-35911) |
| [docs/core/install/windows.md](https://github.com/dotnet/docs/blob/a8768abfe9157bf1aedea468edef3022bc9b4fa7/docs/core/install/windows.md) | [Install .NET on Windows](https://review.learn.microsoft.com/en-us/dotnet/core/install/windows?branch=pr-en-us-35911) |

### CODE REVIEW

**Comment by adegeo** [comment]
@gewarren

File: docs/core/install/includes/verify-download-intro.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+author: adegeo
+ms.author: adegeo
+ms.date: 06/21/2023
+ms.topic: include
+---
+
+After downloading an installer, verify it to make sure that the file hasn't been changed or corrupted. You can verify the checksum on your computer and then compare it to what was reported on the download website.
+
+When downloading an installer or binary from an official download page, the checksum for the file is displayed. Select the **Copy** button to copy the checksum value to your clipboard.
```
Comment (gewarren) [inline docs/core/install/includes/verify-download-intro.md:10]:
```suggestion
When you download an installer or binary from an official download page, the checksum for the file is displayed. Select the **Copy** button to copy the checksum value to your clipboard.
```

Comment (gewarren) [reply docs/core/install/includes/verify-download-intro.md:10]: Acrolinx doesn't like "when downloading..."

File: docs/core/install/includes/verify-download-macos-linux.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Compare the checksum with the value provided by the download site.
+
+> [!IMPORTANT]
+> Even though a Linux file is shown in these examples, this information equally applies to macOS.
+
+### Use a checksum file to validate
+
+The .NET release notes contain a link to a checksum file you can use to validate your downloaded file. The following steps describe how to download the checksum file and validate a .NET install binary:
+
+01. The release notes page for .NET 7.0 on GitHub at <https://github.com/dotnet/core/tree/main/release-notes/7.0> contains a section named **Releases**. The table in this section links to the downloads and checksum files for each .NET 7 release:
```
Comment (gewarren) [inline docs/core/install/includes/verify-download-macos-linux.md:24]:
```suggestion
01. The release notes page for .NET 7 on GitHub at <https://github.com/dotnet/core/tree/main/release-notes/7.0> contains a section named **Releases**. The table in that section links to the downloads and checksum files for each .NET 7 release:
```

File: docs/core/install/includes/verify-download-macos-linux.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+    :::image type="content" source="../media/install-sdk/release-notes-root.png" alt-text="The github release notes version table for .NET":::
+
+01. Select the link to your version of .NET that you downloaded. The previous section used .NET SDK 7.0.304, which is in the .NET 7.0.7 release.
```
Comment (gewarren) [inline docs/core/install/includes/verify-download-macos-linux.md:28]:
```suggestion
01. Select the link for the version of .NET that you downloaded. The previous section used .NET SDK 7.0.304, which is in the .NET 7.0.7 release.
```

File: docs/core/install/includes/verify-download-macos-linux.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+    > [!TIP]
+    > If you're not sure which .NET release contains your checksum file, explore the links until you find it.
```
Comment (gewarren) [inline docs/core/install/includes/verify-download-macos-linux.md:31]: Not sure this is helpful?

File: docs/core/install/includes/verify-download-macos-linux.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+01. In the release page, you can see the .NET Runtime and .NET SDK version, and a link to the checksum file:
+
+    :::image type="content" source="../media/install-sdk/release-notes-version.png" alt-text="The download table with checksums for .NET":::
+
+01. Copy the link to the checksum file.
+
+01. Use the following script, pasting in the link to download the appropriate checksum file:
```
Comment (gewarren) [inline docs/core/install/includes/verify-download-macos-linux.md:39]:
```suggestion
01. Use the following script, but replace the link to download the appropriate checksum file:
```

File: docs/core/install/includes/verify-download-windows.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+648732C79F6276C37A92E211B4C5B6CF84A0A54637C0F85949CED96D31838B43A4DCAE905EF70BAFBC9EDD3542400746FB1E00C4C84679713E97219493A45938
+```
+
+Compare the checksum with the value provided by the download site.
+
+### Use PowerShell and a checksum file to validate
+
+The .NET release notes contain a link to a checksum file you can use to validate your downloaded file. The following steps describe how to download the checksum file and validate a .NET install binary:
+
+01. The release notes page for .NET 7.0 on GitHub at <https://github.com/dotnet/core/tree/main/release-notes/7.0> contains a section named **Releases**. The table in this section links to the downloads and checksum files for each .NET 7 release:
```
Comment (gewarren) [inline docs/core/install/includes/verify-download-windows.md:28]:
```suggestion
01. The release notes page for .NET 7 on GitHub at <https://github.com/dotnet/core/tree/main/release-notes/7.0> contains a section named **Releases**. The table in that section links to the downloads and checksum files for each .NET 7 release:
```

File: docs/core/install/includes/verify-download-windows.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+    :::image type="content" source="../media/install-sdk/release-notes-root.png" alt-text="The github release notes version table for .NET":::
+
+01. Select the link to your version of .NET that you downloaded. The previous section used .NET SDK 7.0.304, which is in the .NET 7.0.7 release.
```
Comment (gewarren) [inline docs/core/install/includes/verify-download-windows.md:32]:
```suggestion
01. Select the link for the version of .NET that you downloaded. The previous section used .NET SDK 7.0.304, which is in the .NET 7.0.7 release.
```

File: docs/core/install/includes/verify-download-windows.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+    > [!TIP]
+    > If you're not sure which .NET release contains your checksum file, explore the links until you find it.
+
+01. In the release page, you can see the .NET Runtime and .NET SDK version, and a link to the checksum file:
+
+    :::image type="content" source="../media/install-sdk/release-notes-version.png" alt-text="The download table with checksums for .NET":::
+
+01. Copy the link to the checksum file.
+
+01. Use the following script, pasting in the link to download the appropriate checksum file:
```
Comment (gewarren) [inline docs/core/install/includes/verify-download-windows.md:43]:
```suggestion
01. Use the following script, but replace the link to download the appropriate checksum file:
```

**Review by gewarren — APPROVED** [review]

File: docs/core/install/includes/verify-download-macos-linux.md (hunk 4 above)
Comment (adegeo) [reply docs/core/install/includes/verify-download-macos-linux.md:31]: Yeah, probably not. There just isn't a one-to-one mapping and I just have a feeling people will arrive there and go "Which one do I download?!"

**Review by leecow — APPROVED** [review]

File: docs/core/install/includes/verify-download-macos-linux.md (hunk 4 above)
Comment (leecow) [reply docs/core/install/includes/verify-download-macos-linux.md]: The answer is clear in the release notes though, interestingly, I am unable to find links to release notes for a given release. Seems like a release notes link on the download pages (e.g. https://dotnet.microsoft.com/en-us/download/dotnet/7.0) would be useful? 

@mairaw - I don't recall if we ever had release notes linked from the download pages? URLs are in releases.json files.

Comment (leecow) [reply docs/core/install/includes/verify-download-macos-linux.md]: You could direct them to the release notes, which makes it fairly clear.

**Pull request merged**